compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -76,6 +77,18 @@ class ScalaSettings extends Settings.SettingGroup {
   val XfatalWarnings: Setting[Boolean] = BooleanSetting("-Xfatal-warnings", "Fail the compilation if there are any warnings.")
   val XverifySignatures: Setting[Boolean] = BooleanSetting("-Xverify-signatures", "Verify generic signatures in generated bytecode.")
 
+  val XmixinForceForwarders = ChoiceSetting(
+    name    = "-Xmixin-force-forwarders",
+    helpArg = "mode",
+    descr   = "Generate forwarder methods in classes inhering concrete methods from traits.",
+    choices = List("true", "junit", "false"),
+    default = "junit")
+
+  object mixinForwarderChoices {
+    def isTruthy(implicit ctx: Context) = XmixinForceForwarders.value == "true"
```

Comment:
isTruthy? What name is that? Use `isTrue` instead?

Reply:
This is copy-pasted from scalac, but I'm fine with changing it.

Reply:
OK, whatever...